dev-workflow-ui/webContent/includes/components/taskDetails/description.xhtml

```diff
@@ -38,7 +38,7 @@
     <jc:user user="#{task.workerUser}" id="workingUser" />
 
     <h:outputText value="Responsible" />
-    <ui:repeat value="#{task.responsibles}" var="responsible">
+    <ui:repeat id="taskResponsibles" value="#{task.responsibles}" var="responsible">
       <jc:user id="taskResponsible" user="#{responsible}" />
```

Comment:
`jc:user` is misleading. It also can be a role. The component should be renamed to `jc:member`.